lib/wrappers/transform-node.js

```diff
@@ -13,6 +14,44 @@ module.exports = class TransformNodeWrapper extends NodeWrapper {
       cachePath: this.cachePath,
     });
     this.callbackObject = this.nodeInfo.getCallbackObject();
+    this.callbackObject.revised = () => {
```

Comment:
nit: can we make this one line?

`this.callbackObject.revised = () => this.revised();`